### ISSUE TITLE
No macOS binary for 0.12.0

### ISSUE THREAD

**Issue by Luflosi** [ipfs-shipyard/ipfs-desktop#1568]
Previous releases had a pre-built binary for macOS but 0.12.0 does not: https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.11.4 vs https://github.com/ipfs-shipyard/ipfs-desktop/releases/tag/v0.12.0.
I guess this is not intentional?

**Comment by jessicaschilling** [comment]
@olizilla and/or @rafaelramalho19 -- can you please triage? Thank you!

**Comment by andrew** [comment]
This also breaks auto-updating for mac as it tries to download `https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.12.0/latest-mac.yml` which doesn't exist, causing this error:

![Screenshot 2020-07-07 at 07 29 35](https://user-images.githubusercontent.com/1060/86727762-a0a86400-c023-11ea-85da-ef85a72411a9.png)

```
    at makeError (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/node_modules/ipfsd-ctl/node_modules/execa/lib/error.js:58:11)
    at handlePromise (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/node_modules/ipfsd-ctl/node_modules/execa/index.js:114:26)
2020-07-07T06:26:39.211Z error: [updater] Error: Cannot find latest-mac.yml in the latest release artifacts (https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.12.0/latest-mac.yml): HttpError: 404 Not Found
"method: GET url: https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.12.0/latest-mac.yml\n\nPlease double check that your authentication token is correct. Due to security reasons actual status maybe not reported, but 404.\n"
Headers: {
  "date": "Tue, 07 Jul 2020 06:26:40 GMT",
  "content-type": "text/plain; charset=utf-8",
  "server": "GitHub.com",
  "status": "404 Not Found",
  "vary": "X-PJAX, Accept-Encoding, Accept, X-Requested-With",
  "cache-control": "no-cache",
  "strict-transport-security": "max-age=31536000; includeSubdomains; preload",
  "x-frame-options": "deny",
  "x-content-type-options": "nosniff",
  "x-xss-protection": "1; mode=block",
  "expect-ct": "max-age=2592000, report-uri=\"https://api.github.com/_private/browser/errors\"",
  "content-security-policy": "default-src 'none'; base-uri 'self'; connect-src 'self'; form-action 'self'; img-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'",
  "content-encoding": "gzip",
  "set-cookie": [
    "_gh_sess=Ue5SrOMIAxE2XLjz1Bzsw26xshzNbPBjhofSuWUYLLmbtx8sUR711IM1O14chXz4dkfoq2fre97n6lkatHcdw2Cm6Nc1e2Sy4kMeyoLj%2B9BjV06QtWpp3ZMMDjsC%2BqnjfPOhbq7ShaEJdQFbm3wYt0Mqj%2BHJ4PJQL2PFtPRGwhpB2jp11WHWKeSEY0GZN9cDyuZiOtfN4WihoksgU8CbOxXe9ilxIy9Smv7xpEN31lWjtCuHKDFbo5y4nrnWo26COvIlAIur74jHbbOfMIQAmw%3D%3D--AbkZ5mKH9aITU5fN--4931oB2e%2FLS1QvE11deg6w%3D%3D; Path=/; HttpOnly; Secure; SameSite=Lax",
    "_octo=GH1.1.2131020528.1594103200; Path=/; Domain=github.com; Expires=Wed, 07 Jul 2021 06:26:40 GMT; Secure; SameSite=Lax",
    "logged_in=no; Path=/; Domain=github.com; Expires=Wed, 07 Jul 2021 06:26:40 GMT; HttpOnly; Secure; SameSite=Lax"
  ],
  "content-length": "40",
  "x-github-request-id": "F830:B839:A47F860:EF09AAF:5F0415A0"
}
    at createHttpError (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/node_modules/builder-util-runtime/out/httpExecutor.js:84:10)
    at ElectronHttpExecutor.handleResponse (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/node_modules/builder-util-runtime/out/httpExecutor.js:169:14)
    at ClientRequest.<anonymous> (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/node_modules/builder-util-runtime/out/httpExecutor.js:134:16)
    at ClientRequest.emit (events.js:210:5)
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:2510:12)
    at SimpleURLLoaderWrapper.emit (events.js:210:5)
2020-07-07T06:27:10.016Z error: [updater] Error: Cannot find latest-mac.yml in the latest release artifacts (https://github.com/ipfs-shipyard/ipfs-desktop/releases/download/v0.12.0/latest-mac.yml): HttpError: 404 Not Found
```

**Comment by lidel** [comment]
This is because notarization failed once again thanks to apple asking us to sign more paperwork:

https://travis-ci.com/github/ipfs-shipyard/ipfs-desktop/jobs/358083565#L533:

> Package Summary:

```
1 package(s) were not uploaded because they had problems:
	/var/folders/17/5mc7816d3mndxjqgplq6057w0000gn/T/480DD798-ADAC-4F5C-A881-3233628711DE/io.ipfs.desktop.itmsp - Error Messages:
		You must first sign the relevant contracts online. (1048)
2020-07-06 22:15:25.262 altool[4207:12202] *** Error: You must first sign the relevant contracts online. (1048)
```

@olizilla are you able to help?
We need to fix this and release v0.12.1 ASAP

**Comment by olizilla** [comment]
More checkboxes have been checked. For posterity, I have to log into both the following sites with Safari and click through the new notification banners.
- https://developer.apple.com/account/ 
- https://appstoreconnect.apple.com/agreements

**Comment by rafaelramalho19** [comment]
Fixed, but you need to upgrade to 0.12.1
